concrete/src/Cache/Page/PageCache.php

```diff
@@ -139,29 +139,30 @@ public function shouldAddToCache(PageView $v)
             return false;
         }
 
-        $app = Application::getFacadeApplication();
-        $u = $app->make(User::class);
-
-        $allowedControllerActions = array('view');
         if (is_object($v->controller)) {
-            if (!in_array($v->controller->getTask(), $allowedControllerActions)) {
+            $allowedControllerActions = array('view');
```

Comment:
This array would make a great constant

Reply:
PHP 5.5 doesn't support arrays as constant names. See https://3v4l.org/PsC9A

Reply:
Maybe Korvin means `[self::CONSTANT_NAME]`?

Reply:
Only @KorvinSzanto can tell... :wink: